lib/doh.h

```diff
@@ -51,7 +54,12 @@ typedef enum {
   DNS_TYPE_NS = 2,
   DNS_TYPE_CNAME = 5,
   DNS_TYPE_AAAA = 28,
+#ifdef USE_HTTPSRR
+  DNS_TYPE_DNAME = 39,           /* RFC6672 */
+  DNS_TYPE_HTTPS = 65
+#else
   DNS_TYPE_DNAME = 39           /* RFC6672 */
+#endif
```

Comment:
I think you can drop these ifdefs, the values can be there always.

Reply:
done